docs/FeatureManagement/Samples/RequirementType.sample.json

```diff
@@ -0,0 +1,144 @@
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Thu, 29 Jun 2023 07:00:00 GMT",
+                                "End": "Wed, 30 Aug 2023 07:00:00 GMT"
+                            }
+                        },
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Tue, 27 Jun 2023 06:00:00 GMT"
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                "id": "FristRequirementTypeAnyFeatureFlag",
+                "description": "Same as Kappa, but requirement type is specified. Will always return true.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Thu, 29 Jun 2023 07:00:00 GMT",
+                                "End": "Wed, 30 Aug 2023 07:00:00 GMT"
+                            }
+                        },
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Tue, 27 Jun 2023 06:00:00 GMT"
+                            }
+                        }
+                    ],
+                    "requirement_type": "Any"
+                }
+            },
+            {
+                "id": "SecondtRequirementTypeAnyFeatureFlag",
+                "description": "Same as Kappa, but requirement type is specified. Will always return true.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Thu, 29 Jun 2023 07:00:00 GMT",
+                                "End": "Wed, 30 Aug 2023 07:00:00 GMT"
+                            }
+                        },
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "End": "Tue, 27 Jun 2023 06:00:00 GMT"
+                            }
+                        }
+                    ],
+                    "requirement_type": "Any"
+                }
+            },
+            {
+                "id": "TrueRequirementTypeAllFeatureFlag",
+                "description": "Requirement type All. Will always return false.",
```

Comment:
The description doesn't match the id.

Reply:
Yeah, I think somewhere in the process the values got switched. Updated.